modules/imgproc/src/fixedpoint.inl.hpp

```diff
@@ -159,7 +161,7 @@ class ufixedpoint64
     static CV_ALWAYS_INLINE ufixedpoint64 one() { return ufixedpoint64((uint64_t)(1ULL << fixedShift)); }
 
     static CV_ALWAYS_INLINE ufixedpoint64 fromRaw(uint64_t v) { return ufixedpoint64(v); }
-    CV_ALWAYS_INLINE ufixedpoint64 raw() { return val; }
+    CV_ALWAYS_INLINE uint64_t raw() { return val; }
```

Comment:
Could you please update `CV_ALWAYS_INLINE ufixedpoint32 raw() { return val; }` and `CV_ALWAYS_INLINE ufixedpoint16 raw() { return val; }` as well